tests/test_pipelines.py

```diff
@@ -17,12 +17,15 @@
 import os
 import random
 import tempfile
+import tracemalloc
 import unittest
 
 import numpy as np
 import torch
 
+import accelerate
```

Comment:
Need to sort the imports to make the linter happy, running isort locally gives this order,

```
import os
import random
import tempfile
import tracemalloc
import unittest

import accelerate
import numpy as np
import PIL
import torch

import transformers
```